.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: python
+sudo: required
+dist: trusty
 python:
   - "3.5"
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install build-essential libssl-dev libffi-dev python-dev
```

Comment:
Was all this needed in the end, wasn't sure if you were trying different approaches to fix the initial cert load problem that is now resolved?